src/app/features/fee-editor/fee-editor.context.ts

```diff
@@ -11,29 +11,27 @@ export interface EditorFee {
   time: string;
 }
 
-export type EditorFees = Record<Exclude<EditorFeeType, 'custom'>, EditorFee> | undefined;
+export type EditorFees = Record<Exclude<EditorFeeType, 'custom'>, EditorFee>;
 
 interface FeeEditorContext {
   availableBalance: Money;
-  customEditorFee: EditorFee | null;
+  customEditorFee: EditorFee;
   customEditorFeeRate: string;
-  currentEditorFee: EditorFee | null;
+  currentEditorFee: EditorFee;
   isLoadingFees: boolean;
   marketData: MarketData;
   editorFees: EditorFees;
-  selectedEditorFee: EditorFee | null;
+  selectedEditorFee: EditorFee;
   getCustomEditorFee(rate: number): EditorFee;
   onSetCustomEditorFeeRate(value: string | null): void;
-  onSetCurrentEditorFee(value: EditorFee | null): void;
-  onSetSelectedEditorFee(value: EditorFee | null): void;
+  onSetCurrentEditorFee(value: EditorFee): void;
+  onSetSelectedEditorFee(value: EditorFee): void;
```

Comment:
From name alone, unclear what the difference between these two is

Reply:
user might select fee type, but not save the selected value and just go back
may be better `onSaveSelectedEditorFee` ?

Reply:
Yeah, the editor view has it's own need to track what the user is selecting, and then once they have 'saved' it that becomes current. We need to track it just in case they cancel. We have a similar need with editing a nonce but we call it `setLoadedNonce`. I'll work on renaming some things with my next pass through the code today.

Reply:
I was able to simplify the state/naming in my last commit. There was just so much going on it was hard to track. I ended up just keeping track of a loaded fee in case the user cancels, but then it seems fine just to keep track of the `selectedFee`.